file/file.go

```diff
@@ -177,6 +179,9 @@ func CreateFolder(path string) error {
 
 // HasStdin determines if the user has piped input
 func HasStdin() bool {
+	if runtime.GOOS == "windows" && (isatty.IsTerminal(os.Stdin.Fd()) || isatty.IsCygwinTerminal(os.Stdin.Fd())) {
```

Comment:
we can use osutils here https://github.com/projectdiscovery/utils/blob/1e40ad2773366a23c63a419611671e63d36a409c/os/os.go#L55